### DISCUSSION TITLE
[lazer] audio offset settings doesn't apply to hitsound effects

### DISCUSSION THREAD

**Discussion by 5ean4cademic** [ppy/osu#15637] [Q&A] [answered]
version: 2021.1111.0-lazer\
description: If you change the offset settings to sync a bluetooth headphone's latency etc. - the offset doesn't apply to the hitsounds and sounds really weird. For both positive and negative offsets regardless of value.

**Comment by bdach** [comment, marked as answer]
If I understand your request correctly, you are asking that hitsounds should respect the audio offset. But if your audio offset is negative, then that means that the hitsound audio should begin playing *before* you hit the key to activate it. Which is impossible, short of an oracle that can predict when you are going to hit the key.

More on this here: https://github.com/ppy/osu/issues/9811#issuecomment-671168153
